http.go

```diff
@@ -64,12 +64,6 @@ func getCommentsHandler(w http.ResponseWriter, r *http.Request) {
 	var err error
 
 	result := resultContainer{Success: true}
-	if r.Method != "POST" {
```

Comment:
+1

We should use a simple/fast router so we can isolate the routes by verb. Julienschmidts is httprouter is the fastest/simplest one.

adtac should approve this PR so I can start on that.